sdk-setup/src/mb2

```diff
@@ -2528,10 +2646,20 @@ fi
 
 remove_wrappers_dir
 
+if [[ -f "CMakeCache.txt" ]]; then
+    cmake_pre_setup
+fi
+
 $cmd "$@"
 
+rc=$?
+
+if [[ -f "CMakeCache.txt" ]]; then
+    cmake_post_cleanup
+fi
+
 ##############################################################################
-exit; fi ###  S E L F - T E S T  EXECUTION BEGINS HERE #######################
+exit $rc; fi ###  S E L F - T E S T  EXECUTION BEGINS HERE ###################
```

Comment:
I would separate the exit on its own line now.

Reply:
fixed